src/relentless/simulate/lammps.py

```diff
@@ -260,6 +264,21 @@ def _call_commands(self, sim):
                     raise ValueError("Angle potential/force is infinite at evaluated r")
             NthetaA = len(thetaA)
 
+        if has_dihedrals:
+            uD = collections.FixedKeyDict(dihedral_type_label.types)
+            fD = collections.FixedKeyDict(dihedral_type_label.types)
+            for i in dihedral_type_label.types:
+                phiD, uD[i], fD[i] = (
+                    sim.potentials.dihedral.linear_space,
+                    sim.potentials.dihedral.energy(key=i),
+                    sim.potentials.dihedral.force(key=i),
+                )
+                if numpy.any(numpy.isinf(uD[i])):
+                    raise ValueError(
+                        "Dihedral potential/force is infinite at evaluated r"
+                    )
+            NphiD = len(phiD)
```

Comment:
Do you need to error check that this has at least 2 elements in it, since you are going to drop one later? We do something like this for the pair potentials, I think.